app/models/sorting.rb

```diff
@@ -23,7 +23,11 @@ class Sorting
   attribute :sort_direction, :string, default: DEFAULT_DIRECTION
 
   def apply_to_scope(scope)
-    scope.order(**order_params)
```

Comment:
I wonder if it would be better to store the names as citext in the database? (We do this with email addresses on review and it works well.)

Reply:
I've used citext much cleaner solution imo